README.md

```diff
@@ -102,3 +102,52 @@ export const writeFileFromText = (text, pathname) => {
   return writeFileSync(join(projectPath, pathname), text, 'utf-8')
 }
 ```
+
+```js
+// converter.test.js
+import { expect } from 'chai'
+import { convertLineToContext, convertLinesToContexts } from './converter';
+import { getTextFromFile } from './file'
+
+describe('test converter', () => {
+  it('Should convert readme to code', () => {
+    const README = getTextFromFile(`./README.md`)
+    const lines = README.split('\n')
+
+    expect(convertLinesToContexts(lines)).is.an('array')
+  })
+  
+  it('Should convert line to context if line has c2e:', () => {
+    const line = 'c2e:src/main.js';
+    expect(convertLineToContext(line).indexOf(getTextFromFile('src/main.js'))).to.not.equal(0)
+  })
+  
+  it('Should return same text if line not have c2e:', () => {
+    const line = '3. Convert them with command';
+    expect(convertLineToContext(line)).to.equal(line)
+  })
+})
+```
+
+```js
+// file.test.js
+import { expect } from 'chai'
+import { getTextFromFile, writeFileFromText } from './file'
+
+describe('test file', () => {
+  it('Should return text of file', () => {
+    const path = 'README.md'
+    expect(getTextFromFile(path)).to.be.a('string')
+  })
+  
+  it('Should write file', () => {
+    const text = 'test';
+    const path = './src/main.js'
+    const backupText = getTextFromFile(path)
+    
+    writeFileFromText(text, path);
```

Comment:
It should be `should write file from text`.
You can generate a simple string and check two things.
- `fs.existsSync('./test/example.md')`
- check the content